AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt

```
@@ -18,1386 +18,478 @@
-    fun filterByFlag(flag: Flag) = launchCatchingTask { viewModel.setFlagFilter(flag) }
-
-    /**
-     * Loads/Reloads (Updates the Q, A & etc) of cards in the [cardIds] list
-     * @param cardIds Card IDs that were changed
-     */
-    private fun updateCardsInList(
-        @Suppress("UNUSED_PARAMETER") cardIds: List<CardId>,
-    ) {
-        updateList()
+    private fun showFilterByTagsDialog() {
     }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)